runLocally.sh

```diff
@@ -169,16 +149,52 @@ function download_menu() {
     }
   fi
 
-  command -v xmllint > /dev/null 2>&1 || {
-    echo >&2 "ERROR: xmllint is not installed. Please install xmllint and retry."
-    exit 1
-  }
-
-  readarray -t region_names <<< "$(xmllint 2>>"$error_log" "$super_region_file" --html --xpath '//tr[@onmouseover]/td[1]/a/text()')"
-  readarray -t region_sub_hrefs <<< "$(xmllint 2>>"$error_log" "$super_region_file" --html --xpath '//tr[@onmouseover]/td[1]/a/@href' | sed 's/.*href="\(.*\)"/\1/')"
-  readarray -t region_osm_hrefs <<< "$(xmllint 2>>"$error_log" "$super_region_file" --html --xpath '//tr[@onmouseover]/td[2]/a/@href' | sed 's/.*href="\(.*\)"/\1/')"
-  readarray -t region_sizes <<< "$(xmllint 2>>"$error_log" "$super_region_file" --html --xpath '//tr[@onmouseover]/td[3]/text()' | sed 's/.*(\(.*\))/\1/')"
+  awk '
```

Comment:
Could you please add a comment above to summarize what this awk should do?

Reply:
Documentation added. Please check again.